deepchem/data/datasets.py

```diff
@@ -420,6 +420,18 @@ def transform(self, transformer: "dc.trans.Transformer", **args) -> "Dataset":
     """
     raise NotImplementedError()
 
+  def select(self, indices: Sequence[int], select_dir: Optional[str] = None):
```

Comment:
There should be a return type of `Dataset` here right? And a "Returns" section in the docstring below

Reply:
I fixed!